### PR TITLE
Add pages describing how to use the app portal

### PR DESCRIPTION
**Pull request by svix-liam**
I added documentation about how to use the App Portal
- Adding Endpoints
- Using the Event Catalog
- Filtering Logs
- Replaying Messages
- Testing event integrations

### CODE REVIEW

File: docs/receiving/using-app-portal/adding-endpoints.mdx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+---
+
+In order to start listening to messages sent through Svix, you will need to configure your **endpoints**.
+
+Adding an endpoint is as simple as providing a URL that you control and a list of **event types** that you want to
+listen to.
+
+![add endpoint](/img/app-portal/add-endpoint.png)
+
+:::info Faster integration with Svix Play
```
Comment (tasn) [inline docs/receiving/using-app-portal/adding-endpoints.mdx:12]: I wouldn't call it integration. More like testing?

**Review by tasn — COMMENTED** [review]
Rotating endpoint secret file is empty

File: docs/receiving/using-app-portal/adding-endpoints.mdx (hunk 1 above)
Comment (svix-liam) [reply docs/receiving/using-app-portal/adding-endpoints.mdx:12]: Users integrate with us

Comment (svix-liam) [reply docs/receiving/using-app-portal/adding-endpoints.mdx]: Reworded for clarity:

`Integrate faster using Svix Play`

**Comment by svix-liam** [comment]
Removed the empty file.

File: docs/receiving/using-app-portal/adding-endpoints.mdx (hunk 1 above)
Comment (svix-liam) [reply docs/receiving/using-app-portal/adding-endpoints.mdx]: sounds good. removed the word integrate.

**Review by tasn — APPROVED** [review]

**Pull request merged**